downloadable-mods.json

```diff
@@ -0,0 +1,109 @@
+        { "repo": "Zurphing/KH1SoraKH3Outfit" },
+        { "repo": "Zurphing/KH1-Keyblades" },
+        { "repo": "TheNja09/DeltaruneSFX" },
+        { "repo": "Mikote111/RecoloredDriveForms" },
+        { "repo": "TheNja09/BlindMode" },
+        { "repo": "TheNja09/CC-Abilities-and-Stats" },
+        { "repo": "SpaceBoiStreams/KH2-PAIN" },
+        { "repo": "SpaceBoiStreams/KH2-Thats-NUTS" },
+        { "repo": "TheNja09/ScoutTF2" },
+        { "repo": "Sai3rina/Steve" },
+        { "repo": "Sai3rina/Alex" },
+        { "repo": "Sai3rina/Female-Voice" },
+        { "repo": "Sai3rina/SoraCap" },
+        { "repo": "Sai3rina/Vivi" },
+        { "repo": "Joe-Degler/Lion-Sora-OpenKH" },
+        { "repo": "camsPatience/KH2-RoxasRFScriptFix" },
+        { "repo": "camsPatience/KH2-RoxasRFProofs-Crown" },
+        { "repo": "SpaceBoiStreams/KH2-Master-Final-Swap" },
+        { "repo": "dallin1016/Vicennial-Trace-Kh2" },
+        { "repo": "Sai3rina/TheGoofyWoody" },
+        { "repo": "Sai3rina/KeybladePack" },
+        { "repo": "dallin1016/Ehrgeiz-Machina-KH2" },
+        { "repo": "Sai3rina/Yuki" },
+        { "repo": "dallin1016/Rebirth-Wing-KH2" },
+        { "repo": "dallin1016/Single-Conflict-KH2" },
+        { "repo": "dallin1016/Lost-in-Phantasm-KH2" },
+        { "repo": "dallin1016/Luminous-Void-KH2" },
+        { "repo": "dallin1016/Materia-Orb-Spell-Icons" },
+        { "repo": "dallin1016/Luminous-Heart-KH2" },
+        { "repo": "SpaceBoiStreams/KH2-Separate-OrgXIII-SpecialBoss-OSTs" },
+        { "repo": "SpaceBoiStreams/KH2-Sonic-Music-Pack" },
+        { "repo": "KHOmega/KH2-SteamDeckButtons" },
+        { "repo": "Sai3rina/ThemeRoyalBlack" },
+        { "repo": "Sai3rina/ThemeLostNobody" },
+        { "repo": "dallin1016/Halloween-Sora-Everywhere" },
+        { "repo": "RaXZerGamingZ/RoxasBTLF_Ledge" },
+        { "repo": "dallin1016/KupoBurst-Moogle" },
+        { "repo": "dallin1016/Scream-Resonance-KH2" },
+        { "repo": "dallin1016/Soul-Resonance-KH2" },
+        { "repo": "Sai3rina/ThemeHeartlessRealm" },
+        { "repo": "Kite2810/Birth-by-Sleep-Keybladepack" },
+        { "repo": "danimalforlife/danima64GLARE" },
+        { "repo": "Sai3rina/CompanionOCs" },
+        { "repo": "danimalforlife/Not-Like-Us" },
+        { "repo": "Sai3rina/LionSoraFixPC" },
+        { "repo": "RaXZerGamingZ/SephirothFF7RQuotes_2FM" }
+    ],
+    "bbs": [
+        { "repo": "Mikote111/VanillaUnversed" },
+        { "repo": "Mikote111/OriginalSFX" },
+        { "repo": "SpaceBoiStreams/KHBBS-Sonic-Music-Pack" },
+        { "repo": "SuperfieldCU/ConsistentLocalizationBBS" },
+        { "repo": "Saro2000/KHBBS-HGSUI" }
+    ],
+    "Recom": [
+        { "repo": "SpaceBoiStreams/KHReCoM-Sonic-Music-Pack" },
+        { "repo": "SuperfieldCU/ConsistentLocalizationRecom" },
+        { "repo": "Saro2000/KHCOM-HGSUI" },
+        { "repo": "EmeraldKeys/DDD-Sora" }
+    ],
+    "kh3d": [
+        { "repo": "IsaacMaGu/DDDtoKH1" },
```

Comment:
This is not a mod to be sued with Mod Manager it only has a readme which tells you to get the file from the releases page of its repo and should be removed